sdk/messaging/azservicebus/internal/go-amqp/.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Binary files (no line-ending conversions), diff using hexdump
```

Comment:
Do we need this file?  Looks like a few others could be pruned too.

Reply:
Yeah, let me go look. I tried to remove a few obvious ones.

Reply:
Got rid of some more files.